Core/src/org/sleuthkit/autopsy/datamodel/ExtractedContent.java

```diff
@@ -154,7 +154,8 @@ static String getIconFilePath(int typeID) {
             return filePath + "encrypted-file.png"; //NON-NLS
         } else if (typeID == BlackboardArtifact.ARTIFACT_TYPE.TSK_EXT_MISMATCH_DETECTED.getTypeID()) {
             return filePath + "mismatch-16.png"; //NON-NLS
-        } else if (typeID == BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_ROUTE.getTypeID()) {
+        } else if (typeID == BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_ROUTE.getTypeID()
+                || typeID == BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_ROUTE.getTypeID()) {
```

Comment:
It looks like this conditional is checking for the same thing. Maybe I'm missing something though

Reply:
you're right.  sorry about that.

Reply:
np!